public/app/features/alerting/unified/components/expressions/ExpressionStatusIndicator.tsx

```diff
@@ -1,18 +1,26 @@
 import { css } from '@emotion/css';
+import { useFormContext } from 'react-hook-form';
 
 import { GrafanaTheme2 } from '@grafana/data';
 import { Badge, clearButtonStyles, useStyles2 } from '@grafana/ui';
 
+import { RuleFormType, RuleFormValues } from '../../types/rule-form';
+
 interface AlertConditionProps {
   isCondition?: boolean;
   onSetCondition?: () => void;
 }
 
 export const ExpressionStatusIndicator = ({ isCondition, onSetCondition }: AlertConditionProps) => {
   const styles = useStyles2(getStyles);
+  const { watch } = useFormContext<RuleFormValues>();
+  const type = watch('type');
+  const isGrafanaRecordingRule = type === RuleFormType.grafanaRecording;
+  const conditionText = isGrafanaRecordingRule ? 'Recording rule condition' : 'Alert condition';
```

Comment:
question: Does it make sense to have conditions for recording rules?

Reply:
The condition is used as the output for the metric. We send this field to the BE side:
```
record: {
          Metric: name, 
          From: condition 
        },
```
We could update the word though. Something like 'output for metric'?

Reply:
i will update to `recording rule output`